apis/ec2/v1beta1/common.go

```diff
@@ -55,6 +55,23 @@ func GenerateEC2Tags(tags []Tag) []ec2.Tag {
 	return res
 }
 
+// EC2TagsNotInCRSpec returns an []ec2.Tag which are present in status.AtProvider and not in spec.ForProvider
+func EC2TagsNotInCRSpec(tags []Tag, ec2Tags []ec2.Tag) []ec2.Tag {
```

Comment:
This function doesn't seem to be used anywhere. I think `DiffEC2Tags` replaces it.

Reply:
yeh a leftover from initial implementation, fixed.